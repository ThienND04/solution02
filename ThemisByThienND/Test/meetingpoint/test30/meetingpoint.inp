421 442 18
284 302 352923
178 334 146029
283 107 385674
138 21 66496
216 55 41416
271 216 469732
59 86 175068
243 61 32763
257 97 137858
366 385 262445
57 337 113046
225 162 309555
40 341 135652
187 45 103180
31 271 914057
137 373 348458
104 329 190511
206 1 846931
18 248 19640
313 249 459879
373 171 40150
258 322 216528
232 199 98269
256 148 409116
8 341 5068
110 50 81627
249 264 232335
267 291 226126
279 215 425397
148 77 13442
174 21 131960
380 403 444507
339 359 217853
260 288 951732
101 75 13465
97 324 391717
328 30 559606
403 288 4671
16 275 349014
209 5 45777
4 240 17410
267 332 198871
220 259 12788
404 23 435892
32 154 255851
250 382 55532
263 387 187798
364 208 393545
315 377 2138
329 351 894645
393 398 70085
153 267 228497
242 76 91217
223 13 4307
83 284 15426
240 403 2791
347 106 22372
52 404 75164
1 274 19699
247 404 265120
253 38 441631
182 316 359747
414 71 457632
150 235 342530
62 230 194624
248 293 451499
94 244 193600
395 54 14038
359 102 46376
252 50 275798
395 308 140734
168 389 105142
412 125 174096
371 181 357166
277 362 2375
274 344 482210
7 10 29708
172 1 667
52 165 82455
303 322 9727
224 307 93478
107 18 1928
226 93 34536
323 364 365588
83 82 97878
77 168 184518
378 7 210470
267 13 706869
226 175 862
179 383 239040
14 228 366522
380 251 101195
362 254 330016
96 270 14076
49 79 127701
345 64 2559
164 272 465238
346 377 34667
14 177 2628
50 233 80664
109 353 405536
365 399 1440
30 263 380152
68 45 38213
13 285 460590
327 143 362664
184 400 432198
298 98 38363
130 419 97252
319 104 92242
336 189 69475
308 106 249760
65 313 524926
411 331 521013
270 413 262048
54 106 94987
235 137 28977
278 350 22443
172 253 3176
141 351 6172
340 234 48835
257 377 268188
89 313 451825
56 142 233742
164 48 385525
291 409 293703
148 131 444644
12 122 345882
310 343 60085
317 55 55411
273 261 8403
50 20 391887
225 1 693319
99 328 61192
33 69 395510
144 93 195651
215 389 302597
391 239 64799
13 313 10330
185 39 13877
68 369 286850
1 145 2468
194 223 3558
248 359 365528
134 106 59445
180 278 125908
319 1 621063
137 76 24621
136 149 235974
196 119 150733
287 194 717790
147 216 113971
207 103 419066
302 138 23009
395 191 78582
43 303 3846
417 267 300303
92 42 162882
124 273 16282
39 50 171698
116 322 442670
374 35 16588
200 276 258655
215 190 431799
294 78 395400
48 356 22128
135 50 167409
66 238 10796
40 128 215880
92 84 115483
45 34 4693
356 64 217293
52 400 465167
210 298 210513
155 52 87319
355 408 21155
145 282 719175
67 66 322077
105 47 384393
20 297 970832
128 6 70501
260 55 525618
220 340 122093
138 9 139472
13 49 414770
375 390 58572
334 46 12185
278 265 138831
166 386 355537
400 269 428448
53 63 454265
193 277 123213
106 351 407212
37 49 421026
175 309 115416
386 49 162380
181 343 132040
74 281 20198
212 163 136340
375 40 138556
141 1 3000
381 388 16930
384 405 13565
375 289 324389
195 111 110943
100 109 515936
253 331 475807
299 286 403100
337 96 161761
57 404 3200
357 244 52639
271 2 405039
289 101 174343
27 163 16490
368 219 380126
202 223 1000000
274 164 7349
182 118 85280
214 106 216218
145 77 529176
215 122 63311
299 342 133857
165 320 370781
172 304 404850
220 236 22708
275 201 20606
47 130 295863
35 361 3628
34 30 361249
185 292 361480
25 261 29573
56 278 88615
348 36 530533
55 262 11364
60 295 236077
106 343 51248
21 414 11505
108 146 96813
167 359 181511
405 237 563063
152 114 355178
207 33 392484
330 134 251871
158 141 29972
123 248 443223
36 370 137295
198 352 240939
194 197 511698
140 358 490815
160 203 126084
78 404 404972
47 77 70563
196 314 291451
20 296 803147
95 148 439841
306 164 550244
368 107 581634
45 316 96118
173 290 140786
365 289 85231
370 136 71225
200 149 69655
392 407 464958
312 61 285139
274 266 687085
191 66 53658
57 61 10386
26 175 558007
353 188 534424
133 299 176412
177 366 132456
18 276 637807
137 90 222978
69 5 74311
196 301 301249
207 139 52899
236 41 248351
99 161 264739
112 353 64102
40 13 399750
172 178 484952
364 49 97296
307 217 100316
214 85 185605
178 114 67740
145 240 4789
300 243 512984
35 135 93339
241 246 207662
397 366 398835
141 140 502506
345 278 145478
91 274 949165
394 414 538854
405 406 678940
250 34 160836
172 384 23237
201 154 158087
111 311 349778
310 13 406300
72 46 67844
231 21 117359
22 18 683329
55 69 7565
378 269 26568
17 409 3249
218 226 50320
78 201 477523
298 26 455539
223 120 25625
185 85 218741
383 113 225792
79 157 365775
407 232 63913
420 43 606976
101 155 108073
163 207 63919
56 40 19464
190 367 713
332 351 897815
409 51 78240
252 305 149708
2 211 211605
317 196 35646
379 226 14100
398 74 323161
109 113 366169
348 227 67679
248 239 487209
172 402 971939
326 280 107990
269 241 16713
204 148 134044
326 45 205089
160 112 15660
80 404 64098
69 59 117721
366 102 10536
145 36 397297
176 251 123974
49 183 542169
407 400 422137
31 226 336274
53 210 240880
8 151 452018
241 111 24607
240 408 4704
253 234 657311
29 316 224159
97 77 25655
215 4 494040
271 189 829279
376 33 924352
400 351 19868
194 60 579037
169 106 207576
17 167 53157
310 286 20446
316 125 330309
206 215 328222
268 331 229789
20 88 664339
325 294 206032
321 55 493374
220 163 93735
360 284 37517
1 33 37757
275 127 10169
44 109 921979
313 26 19853
110 159 331720
78 223 8517
194 303 391508
391 28 352418
212 187 126450
78 30 20856
326 270 105171
102 349 482583
11 49 508691
274 19 595075
205 250 171820
255 40 12172
308 15 177873
384 173 564916
70 375 106470
271 313 940
207 58 41994
156 149 135403
396 351 481030
232 362 58605
2 414 25013
40 143 138198
217 106 14796
207 333 232118
108 79 370729
372 253 397109
87 129 55866
387 416 373191
401 184 208796
109 240 24186
265 410 186825
147 73 320285
1 13 1247
217 229 102222
221 18 675630
359 74 126929
193 186 152375
25 255 524269
357 277 96322
59 222 41014
8 115 403410
363 415 54784
213 172 663076
381 403 416264
110 318 495771
69 170 269426
304 76 179980
87 351 460194
20 271 9484
263 331 64569
198 170 11307
245 232 136263
418 66 312480
192 240 572216
223 107 9968
373 421 53996
373 310 551030
81 295 20612
24 240 440139
354 183 25730
92 145 475558
363 281 383314
35 338 264897
132 269 69888
3 370 440846
192 126 257937
121 136 395447
362 135 7601
194 230 415504
53 50 42713
335 97 161175
275 117 484121
5412 867174
985284 193117
76654 195109
38199 590548
201592 629676
899573 935101
804471 835376
143266 671409
58501 775985
763428 70713
723027 773417
24467 603518
966606 392199
257618 951957
416398 455734
426436 702693
346413 160267
613076 405196
