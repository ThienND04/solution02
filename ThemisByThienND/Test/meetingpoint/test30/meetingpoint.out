5480039664
195304243142
73478684704
36616644624
193241252992
526697292781
471015357087
137331349216
56077654576
71513895438
423330139419
24774599124
391975165302
246946431968
243799779806
282801608622
140476360521
248612740292
